src/node_api.cc

```diff
@@ -2148,12 +2148,12 @@ napi_status napi_get_value_int64(napi_env env,
   // v8::Value::IntegerValue() converts NaN to INT64_MIN, inconsistent with
   // v8::Value::Int32Value() that converts NaN to 0. So special-case NaN here.
   double doubleValue = val.As<v8::Number>()->Value();
-  if (std::isnan(doubleValue)) {
-    *result = 0;
-  } else {
+  if (std::isfinite(doubleValue)) {
```

Comment:
Can you update the comment above as well.  It refers to NaN and the check no longer does.  Would be good if the comment explained why the broader check makes sense.

Reply:
Yup, I forgot to do that.